webviews/components/header.tsx

```diff
@@ -66,50 +67,66 @@ function Title({ title, number, url, canEdit, isCurrentlyCheckedOut, isIssue, re
 				}
 			}}
 		>
-			<textarea name="text" style={{ width: '100%' }} defaultValue={currentTitle}></textarea>
+			<input type="text" style={{ width: '100%' }} defaultValue={title}></input>
 			<div className="form-actions">
 				<button className="secondary" onClick={() => setEditMode(false)}>
 					Cancel
 				</button>
 				<input type="submit" value="Update" />
 			</div>
 		</form>
-	) : (
-		<h2>
-			{currentTitle} <a href={url} title={url}>#{number}</a>
-		</h2>
 	);
 
-	return (
+	const displayTitle = (
 		<div className="overview-title">
-			<div className="title-and-edit">
-				{editableTitle}
-				<div className="block-select">
-					{/*
-				For whatever reason, triple click on a block element in MacOS will select everything in that element, *and* every `user-select: false` block adjacent to that element.
-				Add an empty selectable div here to block triple click on title from selecting the following buttons. Issue #628.
-				*/}
-				</div>
-				{canEdit && !inEditMode ? (
-					<div className="flex-action-bar comment-actions">
-						{
-							<button title="Edit" onClick={() => setEditMode(true)}>
-								{editIcon}
-							</button>
-						}
-						{
-							<button title="Copy Link" onClick={copyPrLink}>
-								{copyIcon}
-							</button>
-						}
+			<h2>
+				{title}{' '}
+				<a href={url} title={url}>
+					#{number}
+				</a>
+			</h2>
+		</div>
+	);
+
+	const editableTitle = inEditMode ? titleForm : displayTitle;
+	return editableTitle;
+}
+
+function ButtonGroup({ isCurrentlyCheckedOut, canEdit, isIssue, repositoryDefaultBranch, setEditMode }) {
+	const { refresh, copyPrLink } = useContext(PullRequestContext);
+
+	return (
+		<div className="button-group">
+			<CheckoutButtons {...{ isCurrentlyCheckedOut, isIssue, repositoryDefaultBranch }} />
+			<button onClick={refresh} className="secondary">
+				Refresh
+			</button>
+			{canEdit && (
+				<>
+					<button title="Rename" onClick={setEditMode} className="secondary">
+						Rename
+					</button>
+					<button title="Copy Link" onClick={copyPrLink} className="secondary">
+						Copy Link
+					</button>
+				</>
+			)}
+		</div>
+	);
+}
+
+function Subtitle({ state, isDraft, isIssue, author, base, head }) {
+	return (
+		<div className="subtitle">
+			<div id="status">{getStatus(state, isDraft)}</div>
+			<div className="author">
+				{!isIssue ? <Avatar for={author} /> : null}
+				{!isIssue ? (
+					<div className="merge-branches">
+						<AuthorLink for={author} /> {getActionText(state)} into{' '}
+						<code className="branch-tag"> {base}</code> from <code className="branch-tag"> {head} </code>
```

Comment:
Not sure about the spacing here. Is there a difference when there's a space in the code tag and a space outside? And, does the space after `{head}` add padding?

Reply:
Seems like it had no effect on render but I cleaned it out anyway.